docs/src/recipes/appendix.asciidoc

```diff
@@ -0,0 +1,226 @@
+////
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+////
+Appendix
+========
+
+Many of the recipes are based on questions and answers provided on the gremlin-users mailing list. This section
```

Comment:
I debated on whether to do that or not with recipes in general but given that I usually adapt the traversals to a narrative, alter the data or improve upon them from a performance perspective or usability perspective linking back doesn't always make sense, so rather than do it some of the times, I opted to not do it at all. The "appendix" might be different though....I don't recall if I made any modifications to those or just "copied" them writing my own explanation for what they were doing as a "title". Maybe adding the links for those would be useful in some way....let me think on that.